docs/api.md

```diff
@@ -266,6 +267,40 @@ Deletes a package domain.
 |---------|------------------------------|----------|
 | name    | The full name of the domain  | No       |
 
+### Python Packages
+
+#### List package domains
+`GET https://scarf.sh/api/v1/packages/{package-id}/domains`
+
+List all package domains for the user.
```

Comment:
@fabioluz do we define what a `package domain` is anywhere? If not, we should probably make sure it's explicitly defined and link to that definition perhaps.

Reply:
We do here, in the same way we do for other package types
![image](https://user-images.githubusercontent.com/5049361/158443110-478c3d82-0326-429e-b6a2-e43cf978b132.png)

Reply:
Note that is in the Gateway Docs not in the API Docs

Reply:
Ok, let's just link to that section here.